test/integration/targets/groupby_filter/runme.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Unfortunately, I cannot use `-u`, I had that there originally.  It results in a failure when attempting to activate a virtualenv, with the following error:

```
bash: _OLD_VIRTUAL_PATH: unbound variable
```

There is apparently a long history as documented at https://github.com/pypa/virtualenv/issues/150

I started to try defining the missing vars, and stopped at 5 variables.